pkgs/by-name/te/technitium-dns-server/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  makeWrapper,
+  dotnet-sdk_8,
+  ...
```

Comment:
```suggestion
```

Reply:
Done. Removed the , after the last package as well.